solr/solrj-streaming/build.gradle

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: 'java-library'
+
+description = 'Solrj-Streaming - SolrJ requiring Streaming Expressions'
+
+dependencies {
+
+
+    implementation project(':solr:solrj')
+
+    // declare dependencies we use even though already declared by solrj-core
+    implementation 'org.slf4j:slf4j-api'
+    implementation 'org.apache.httpcomponents:httpclient'
+    implementation 'org.apache.httpcomponents:httpcore'
+    implementation 'org.apache.commons:commons-math3'
+
+    testImplementation project(':solr:test-framework')
+    testImplementation project(':solr:core')
+
+    testImplementation 'junit:junit'
+    testImplementation 'commons-io:commons-io'
+    testImplementation 'com.google.guava:guava'
+    testImplementation project(':solr:solrj-zookeeper')
+    testImplementation('org.apache.zookeeper:zookeeper', {
+        exclude group: "org.apache.yetus", module: "audience-annotations"
+    })
+
+    permitTestUsedUndeclared project(':solr:solrj-zookeeper') // duh!
```

Comment:
nit: can you put this line under `testImplementation project(':solr:solrj-zookeeper')` - I've been trying to keep the `permitTestUsedUndeclared` near the line that introduced it.

Reply:
sure